custom_components/thermal_comfort/sensor.py

```diff
@@ -299,30 +311,38 @@ def id_generator(unique_id: str, sensor_type: str) -> str:
     return unique_id + sensor_type
 
 
-class SensorThermalComfortCommon(SensorEntity):
+class SensorThermalComfort(SensorEntity):
     """Representation of a Thermal Comfort Sensor."""
 
     def __init__(
         self,
-        device,
-        entity_description,
-        icon_template,
-        entity_picture_template,
-        sensor_type,
-        unique_id,
-    ):
+        device: "DeviceThermalComfort",
+        sensor_type: SensorType,
+        entity_description: SensorEntityDescription,
+        device_name: str = None,
+        icon_template: Template = None,
+        entity_picture_template: Template = None,
+        device_unique_id: str = None,
+    ) -> None:
         """Initialize the sensor."""
-        self._device = device
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, device_unique_id)},
+        )
```

Comment:
Should `name` be dropped while merging classes back?

Reply:
No my bad, thanks for pointing that out